src/dlgTriggerEditor.cpp

```diff
@@ -8370,15 +8365,50 @@ void dlgTriggerEditor::resizeEvent(QResizeEvent* event)
 void dlgTriggerEditor::slot_key_grab()
 {
     mIsGrabKey = true;
+    setShortcuts(true);
+    QCoreApplication::instance()->installEventFilter(this);
+}
+
+// Activate shortcuts for editor menu items like Ctrl+S for "Save Item" etc.
+// Deactivate instead with optional "true" - useful to grab for a keybinding
+void dlgTriggerEditor::setShortcuts(const bool unsetInstead)
+{
     QList<QAction*> actionList = toolBar->actions();
+    QString actionText;
     for (auto& action : actionList) {
-        if (action->text() == "Save Item") {
-            action->setShortcut(tr(""));
-        } else if (action->text() == "Save Profile") {
-            action->setShortcut(tr(""));
+        actionText = action->text();
+        if (actionText ==  tr("Save Item")) {
+            action->setShortcut((unsetInstead) ? tr("") : tr("Ctrl+S"));
```

Comment:
One should not translate nothing, I think you should be using:
```suggestion
            action->setShortcut((unsetInstead) ? QString() : tr("Ctrl+S"));
```

Reply:
It was using "" before. I did not change that.